vscode-antimony/src/server/stibium/test/import-test-data/import.ant

```diff
@@ -0,0 +1,14 @@
+a = 3
```

Comment:
I would include an 'import_bad.ant' file, too, with some error in it, so you can test importing a bad file and make sure you give out a coherent error message.

Reply:
Added bad import test case